problem_builder/step.py

```diff
@@ -121,11 +121,27 @@ def allowed_nested_blocks(self):
         NestedXBlockSpec allows explicitly setting disabled/enabled state, disabled reason (if any) and single/multiple
         instances
         """
+        additional_blocks = []
+        try:
+            from xmodule.video_module.video_module import VideoDescriptor
+            additional_blocks.append(NestedXBlockSpec(
+                VideoDescriptor, category='video', label=_(u"Video")
+            ))
+        except ImportError:
+            pass
+        try:
+            from imagemodal import ImageModal
+            additional_blocks.append(NestedXBlockSpec(
+                ImageModal, category='imagemodal', label=_(u"Image Modal")
+            ))
+        except ImportError:
+            pass
+
         return [
             NestedXBlockSpec(AnswerBlock, boilerplate='studio_default'),
             MCQBlock, RatingBlock, MRQBlock, HtmlBlockShim,
             AnswerRecapBlock, MentoringTableBlock,
-        ]
+        ] + additional_blocks
```

Comment:
@Kelketek Now that this method does more than simply returns a static list, do you think it would make sense to add a unit test for it? I'm not sure how easy it is, but I'm guessing you could [fake the modules](http://stackoverflow.com/questions/5122465/can-i-fake-a-package-or-at-least-a-module-in-python-for-testing-purposes) that contain `VideoDescriptor` and `ImageModal` in tests.